cli/smartcontract/smart_contract.go

```diff
@@ -98,30 +106,46 @@ func NewCommands() []*cli.Command {
 	invokeFunctionFlags = append(invokeFunctionFlags, options.RPC...)
 	deployFlags := append(invokeFunctionFlags, []cli.Flag{
 		&cli.StringFlag{
-			Name:    "in",
-			Aliases: []string{"i"},
-			Usage:   "Input file for the smart contract (*.nef)",
+			Name:     "in",
+			Aliases:  []string{"i"},
+			Required: true,
+			Usage:    "Input file for the smart contract (*.nef)",
+			Action: func(c *cli.Context, value string) error {
+				if value == "" {
+					return cli.Exit(fmt.Errorf("the i flag requires a non-empty value"), 1)
+				}
+				return nil
+			},
 		},
 		&cli.StringFlag{
-			Name:    "manifest",
-			Aliases: []string{"m"},
-			Usage:   "Manifest input file (*.manifest.json)",
+			Name:     "manifest",
+			Aliases:  []string{"m"},
+			Required: true,
+			Usage:    "Manifest input file (*.manifest.json)",
+			Action: func(c *cli.Context, value string) error {
+				if value == "" {
+					return cli.Exit(fmt.Errorf("the m flag requires a non-empty value"), 1)
+				}
+				return nil
+			},
 		},
 	}...)
 	manifestAddGroupFlags := append([]cli.Flag{
 		&cli.StringFlag{
-			Name:    "sender",
-			Aliases: []string{"s"},
-			Usage:   "deploy transaction sender",
+			Name:     "sender",
+			Aliases:  []string{"s"},
+			Required: true,
```

Comment:
Always use action for required flags to check that flag is not empty. Otherwise it's useless.

Reply:
Check it everywhere.